mapper/complementaryContentMapper.go

```diff
@@ -65,7 +73,7 @@ func (ccm *ComplementaryContentCPHMapper) mapToUppComplementaryContentDelete(mpc
 	}
 }
 
-func buildCCAlternativeTitles(promoTitle string) *model.AlternativeTitles {
+func (ccm *ComplementaryContentCPHMapper) buildCCAlternativeTitles(promoTitle string) *model.AlternativeTitles {
```

Comment:
What is the purpose of these function having a `ComplementaryContentCPHMapper` receiver?

Reply:
this doesn't use `ccm` but it's similar to `buildCCAlternativeImages`, and it is anyways part of the mapper to build these attributes so the responsibility falls on this struct.